src/pages/help-centre/deposits-and-withdrawals.js

```diff
@@ -109,6 +76,7 @@ const LiftWithdrawalLimits = () => (
                 components={[
                     <strong key={0} />,
                     <ExternalLink
+                        to=""
```

Comment:
any reason for blank to?

Reply:
`LocalizedLink` has to have a to property even if it's not used. It will cause an error like [this](https://trello-attachments.s3.amazonaws.com/5a06b75c870324d38d94e579/5fcdc5599e4490465d65731c/c23f7cb2563dc9844807be17ec16837a/Screen_Recording_2020-12-15_at_5.49.36_PM.mov) if we omit it.